tests/check_consistent.py

```diff
@@ -115,21 +115,29 @@ def check_versions():
         module = m.group(1)
         assert module not in versions, f"Duplicate module {module} in VERSIONS"
         versions.add(module)
-    modules = set()
-    for entry in os.listdir("stdlib"):
-        if entry == "@python2" or entry == "VERSIONS":
-            continue
-        if os.path.isfile(os.path.join("stdlib", entry)):
-            mod, _ = os.path.splitext(entry)
-            modules.add(mod)
-        else:
-            modules.add(entry)
-    extra = modules - versions
+    modules = _find_stdlib_modules()
+    # Sub-modules don't need to be listed in VERSIONS.
+    extra = {m.split(".")[0] for m in modules} - versions
     assert not extra, f"Modules not in versions: {extra}"
     extra = versions - modules
     assert not extra, f"Versions not in modules: {extra}"
 
 
+def _find_stdlib_modules() -> set[str]:
+    modules = set()
+    for path, _, files in os.walk("stdlib"):
+        if "@python2" in path:
```

Comment:
This is a bit of a hack, although likely not a problem in practice.

Reply:
At some point we could extend this to handle Python 2 modules as well. It could even check if Python 2 modules are all marked as `2.7-` and only those.

Reply:
But then again, I don't want to spend considerable amount of energy on Python 2, and those modules are unlikely to change.